File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        int n = nums.size(),i,j,fix,flag = 0;
        for(i=0;i<n;i++){
            fix=nums[i];
            for(j=i+1;j<n;j++){
                if(fix==nums[j]){
                    flag=1;
                    break;
                }
            }
            if(flag == 1) break;
        }
        if(flag==1)
            return 1;
        else
            return 0;
    }
};

File: Data Structures & Algorithms/majority-element/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        if not nums: return None
        freq = Counter(nums)
        most_common_element, max_count = freq.most_common(1)[0]
        if max_count > len(nums)/2:
            return most_common_element
        return None
        